proto/wechaty/puppet/room.proto

```diff
@@ -27,6 +27,7 @@ message RoomPayloadResponse {
 message RoomAddRequest {
   string id = 1;
   string contact_id = 2;
+  string force_invite = 3;
```

Comment:
Could you please make sure that the type `string` is what we expected?

I think it should be `boolean`?